tests/Network/NetworkTest.php

```diff
@@ -200,6 +200,28 @@ public function testFactoryPresets()
 
         $this->assertEquals('2Mwx4ckFK9pLBeknxCZt17tajwBEQXxNaWV', $p2sh->getAddress(NetworkFactory::dogecoinTestnet()));
         $this->assertEquals('nUpssuonax8qjuc3zU3cwmE9n9W7QXJjgW', $p2pk->getAddress(NetworkFactory::dogecoinTestnet()));
+
+        // Dash
+        $this->assertEquals(NetworkFactory::dash()->getAddressByte(), '4c');
+        $this->assertEquals(NetworkFactory::dash()->getP2shByte(), '10');
+        $this->assertEquals(NetworkFactory::dash()->getPrivByte(), 'cc');
+        $this->assertEquals(NetworkFactory::dash()->isTestnet(), false);
+        $this->assertEquals(NetworkFactory::dash()->getHDPrivByte(), '02fe52cc');
+        $this->assertEquals(NetworkFactory::dash()->getHDPubByte(), '02fe52f8');
+        $this->assertEquals(NetworkFactory::dash()->getNetMagicBytes(), 'bd6b0cbf');
+        $this->assertEquals('7X7VPCbTMLvUSjhMo3vdqKb8eNrccxgkJ1', $p2sh->getAddress(NetworkFactory::dash()));
+        $this->assertEquals('XbKZStn8KGzRUsSr5wiq18A3VUyD7pdKXX', $p2pk->getAddress(NetworkFactory::dash()));
+
+        // Dash testnet
+        $this->assertEquals(NetworkFactory::dashTestnet()->getAddressByte(), '8b');
+        $this->assertEquals(NetworkFactory::dashTestnet()->getP2shByte(), '13');
+        $this->assertEquals(NetworkFactory::dashTestnet()->getPrivByte(), 'ef');
+        $this->assertEquals(NetworkFactory::dashTestnet()->isTestnet(), true);
```

Comment:
https://travis-ci.org/Bit-Wasp/bitcoin-php/builds/105893301

Get this test working by adding a 4th param `true` in the `NetworkFactory::dashTestnet()`